cli_reference/opm/cli-opm-ref.adoc

```diff
@@ -19,9 +19,12 @@ $ opm <command> [<subcommand>] [<argument>] [<flags>]
 |===
 |Flag |Description
 
-|`--skip-tls`
+|`--skip-tls-verify`
 |Skip TLS certificate verification for container image registries while pulling bundles or indexes.
 
+|`--use-http`
+|Use basic HTTP authentication for container image registries while pulling bundles.
```

Comment:
Flag changes were downstreamed in https://github.com/openshift/operator-framework-olm/pull/269
`skip-tls` has been deprecated:

```shell
╰─ ./bin/opm migrate --skip-tls
Flag --skip-tls has been deprecated, use --use-http and --skip-tls-verify instead
```

Reply:
Should the opm migrate command have a deprecated blob like `opm index` does?  It feels heavy-handed to have this for every SQLite-related command, but `opm migrate` definitely is one, so it feels like maybe some general notation/brief message would be handy. ??

Reply:
@grokspawn My DPM confirmed that since `--skip-tls` has been deprecated and replaced, updates to the documentation of these flags needs to go through change management.

As a result, I am going to create a separate ticket and PR for the `--skip-tls` deprecation and associated replacement flags. That way, the `opm migrate` change is not held up while I wrangle the additional acks required for change management.

Reply:
Here is the ticket for the deprecated `--skip-tls` update: https://issues.redhat.com/browse/OCPBUGS-8255